Makefile

```diff
@@ -18,7 +18,7 @@ WEBROOT:=`pwd`/frontends/web
 catch:
 	@echo "Choose a make target."
 envinit:
-	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.19.1
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b ${GOPATH}/bin v1.19.1
```

Comment:
This is strange. Both should behave exactly the same:

> To substitute a variable’s value, write a dollar sign followed by the name of the variable in parentheses or braces: either ‘$(foo)’ or ‘${foo}’ is a valid reference to the variable foo

(from https://www.gnu.org/software/make/manual/html_node/Reference.html#Reference)

Or is it not GNU make? Either way, I think POSIX make should be the same. What's your `make --version`?

But we really should update to go1.14 and use the standard go install method: https://github.com/golangci/golangci-lint#go

Reply:
That probably depends on the shell, and on macOS I get the following behaviour:

```
$ export FOO=bar
$ echo $FOO
bar
$ echo ${FOO}
bar
$ echo $(FOO)
-bash: FOO: command not found
```

Reply:
Could you try this simple makefile:

```
$ cat Makefile
dummy:
	@echo ${SHELL}
```

And also, what does `make --version` say?

Reply:
Ah, the errors I had might not be directly related, if GNU make handles variable substitution independently from the shell.

Reply:
Fixed this issue by moving the variables PATH and GOPATH to the top of the `Makefile`